src/Web/Web.csproj

```diff
@@ -14,41 +14,44 @@
 		<ImplicitUsings>enable</ImplicitUsings>
 		<Nullable>enable</Nullable>
 		<SpaRoot>ClientApp</SpaRoot>
+		<SpaProxyServerUrl>http://localhost:44467</SpaProxyServerUrl>
+		<SpaProxyLaunchCommand>npm start</SpaProxyLaunchCommand>
 	</PropertyGroup>
 
 	<ItemGroup>
         <!-- Don't publish the SPA source files, but do show them in the project files list -->
-        <Content Remove="$(SpaRoot)\**"/>
-        <None Remove="$(SpaRoot)\**"/>
-        <None Include="$(SpaRoot)\**" Exclude="$(SpaRoot)\node_modules\**"/>
+        <Content Remove="$(SpaRoot)\**" />
+        <None Remove="$(SpaRoot)\**" />
+        <None Include="$(SpaRoot)\**" Exclude="$(SpaRoot)\node_modules\**" />
     </ItemGroup>
 
 	<Target Name="NpmInstall" BeforeTargets="Build" Condition="!Exists('$(SpaRoot)\node_modules')">
         <!-- Ensure Node.js is installed -->
         <Exec Command="node --version" ContinueOnError="true" EchoOff="true">
-            <Output TaskParameter="ExitCode" PropertyName="ErrorCode"/>
+            <Output TaskParameter="ExitCode" PropertyName="ErrorCode" />
         </Exec>
-        <Error Condition="'$(ErrorCode)' != '0'" Text="Node.js is required to build and run this project. To continue, please install Node.js from https://nodejs.org/, and then restart your command prompt or IDE."/>
-        <Message Importance="high" Text="Restoring dependencies using 'npm'. This may take several minutes..."/>
-        <Exec WorkingDirectory="$(SpaRoot)\" Command="npm install"/>
+        <Error Condition="'$(ErrorCode)' != '0'" Text="Node.js is required to build and run this project. To continue, please install Node.js from https://nodejs.org/, and then restart your command prompt or IDE." />
+        <Message Importance="high" Text="Restoring dependencies using 'npm'. This may take several minutes..." />
+        <Exec WorkingDirectory="$(SpaRoot)\" Command="npm install" />
     </Target>
 
     <Target Name="NpmRunBuild" BeforeTargets="Build" DependsOnTargets="NpmInstall">
-        <Exec WorkingDirectory="$(SpaRoot)\" Command="npm run build"/>
+        <Exec WorkingDirectory="$(SpaRoot)\" Command="npm run build" />
```

Comment:
Hmm... Should we remove this since we only care about running `npm run build` during a `dotnet pack`? This might be the reason why `dotnet run` still waits for `ng build` to finish.